slitlessutils/examples/wr96.py

```diff
@@ -93,13 +109,7 @@ def preprocess_direct():
         files.append(imgfile)
 
     # mosaic data via astrodrizzle
-    astrodrizzle.AstroDrizzle(files, output=ROOT, build=False,
-                              static=False, skysub=True, driz_separate=False,
-                              median=False, blot=False, driz_cr=False,
-                              driz_combine=True, final_wcs=True,
-                              final_rot=0., final_scale=SCALE,
-                              final_pixfrac=1.0,
-                              overwrite=True, final_fillval=0.0)
+    drizzle(files, output=ROOT, **WR96_DRIZZLE_PARAMS)
```

Comment:
I don't think this is a good idea to use your function here. Because your thing will be fine-tuned to cosmic ray clipping. And the purpose here is not for CR rej. So, can you make it back to the other way?

Reply:
Reverted here: https://github.com/spacetelescope/slitlessutils/pull/27/commits/1d61459b168e134a749c021a1c6b552bf26ecb62